notebooks/ch15/bias_cv.md

```diff
@@ -0,0 +1,34 @@
+
+## Cross-Validation
+
+In the previous section, we observe that the we need a way to simulate test error to more accurately manage the bias-variance trade off. Cross-validation provides a method of estimating our model error using a single observed dataset by separating data used for training from the data used for model selection and final accuracy. 
+
+## K-Fold Cross-Validation
+
+In Chapter 14.3, we mentioned the method of the **train-validation split**, which can be used to simulate test error through the validation set. However, this method is prone to high variance of the validation error because the evaluation of the error may depend heavily on which points end up in the training and validation sets.
```

Comment:
First sentence: "In Chapter 14.3, we used the **train-validation split** method to simulate test error through the validation set." (Maybe also include some sentence or short note that connects train-validation and train-validation-test split since we only mention the latter in 14.3. As is, 14.3 never mentions simulating test error with the validation set since it only mentions feature selection.)
Second sentence: "However, this method is prone to high variance because the evaluation of the [validation] error may depend heavily on which points end up in the training and validation sets."

Reply:
want to leave second sentence as is because we need to make it clear the it's the variance of the validation error, not model variance

Reply:
how would you suggest linking them?

Reply:
Maybe for the second sentence: "However, with this method the validation error may be prone to high variance because the evaluation of the error may depend heavily on which points end up in the training and validation sets."

Linking: On second thought, maybe this is not really necessary since the two are pretty closely related.